lib/readline.js

```diff
@@ -522,8 +522,7 @@ function handleGroup(self, group, width, maxColumns) {
       var item = group[idx];
       self._writeToOutput(item);
       if (col < maxColumns - 1) {
-        for (var s = 0, itemLen = item.length; s < width - itemLen;
-             s++) {
+        for (var s = 0, itemLen = item.length; s < width - itemLen; s++) {
```

Comment:
By the way, this could be simplified to `for (var s = 0; s < width - item.length; s++)`, V8 should be able to handle that.